### PR TITLE
Validation logic to wait until the consul port 8500 is open before starting mft (issue #128)

### PR DESCRIPTION
**Pull request by ShivangMishra**
With this change, issue #128 is resolved.
image

### CODE REVIEW

**Comment by ShivangMishra** [comment]
@smarru @DImuthuUpe could you please review and merge this PR?

File: python-cli/mft_cli/airavata_mft_cli/bootstrap.py [hunk 1]
```diff
@@ -160,6 +164,10 @@ def start_mft():
     zip_path = os.path.join(os.path.expanduser('~'), ".mft/Standalone-Service-0.01-bin.zip")
     download_and_unarchive(url, zip_path)
 
+  while not is_port_open(8500):
+    print("Waiting for Consul to start...")
+    time.sleep(1)
```
Comment (DImuthuUpe) [inline python-cli/mft_cli/airavata_mft_cli/bootstrap.py:169]: @ShivangMishra Can you make this only run for 20 seconds and if the port is not open by then, throw an error message and exit the program?

**Review by DImuthuUpe — CHANGES_REQUESTED** [review]
Looks promising. Can you make the wait loop bounded to 20 seconds?

File: python-cli/mft_cli/airavata_mft_cli/bootstrap.py (hunk 1 above)
Comment (ShivangMishra) [reply python-cli/mft_cli/airavata_mft_cli/bootstrap.py:172]: Yes sure. Done.

**Comment by DImuthuUpe** [comment]
Looks great. Thanks @ShivangMishra

**Pull request merged**